### DISCUSSION TITLE
adding URL to each country?

### DISCUSSION THREAD

**Discussion by vahidtakro** [themustafaomar/jsvectormap#45] [Ideas]
Hello, can I add url for each country? or make a country in different color? or add a flag css on it?
I want it for my travel blog.

**Comment by themustafaomar** [comment]
Initially, you can't add a URL for each country but you can have an object that contains all the countries' keys, and when someone clicks|(or any other event) the region you can check if this region has a URL and do whatever you want, as in the pen below.

Image flag can be done using markers check the example below.

https://codepen.io/themustafaomar/pen/abWQbeQ

About different colors, currently you can't add different colors to each region without using `series`

**Reply by vahidtakro** [reply]
Thanks, I'll check this out.
